### PR TITLE
chore(SNP-738) Package release

### PR DESCRIPTION
**Pull request by martynov-alex**
Added files necessary for package release.

### CODE REVIEW

File: example/pubspec.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+environment:
+  sdk: '>=3.1.2 <4.0.0'
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_localizations:
+    sdk: flutter
+  intl: ^0.18.1
+  mocktail: ^1.0.1
```
Comment (maxim-sysoev) [inline example/pubspec.yaml:17]: this is should be dev dependency

File: example/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  provider: ^6.1.1
+  surf_widget_test_composer:
```
Comment (maxim-sysoev) [inline example/pubspec.yaml:19]: This is also dev dependency

File: pubspec.yaml [hunk 3]
```diff
@@ -1,21 +1,22 @@
 name: surf_widget_test_composer
-description: A new Flutter package project.
+description: Utility that simplifies widget and golden testing.
```
Comment (maxim-sysoev) [inline pubspec.yaml:2]: this library only for golden testing, not so?
I guess we should change package name

File: example/pubspec.yaml (hunk 1 above)
Comment (martynov-alex) [reply example/pubspec.yaml:17]: fixed

File: example/pubspec.yaml (hunk 2 above)
Comment (martynov-alex) [reply example/pubspec.yaml:19]: fixed

File: pubspec.yaml [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
   flutter: ">=1.17.0"
 
 dependencies:
   flutter:
     sdk: flutter
-
-  golden_toolkit: ^0.15.0
   flutter_test:
     sdk: flutter
+  golden_toolkit: ^0.15.0
```
[the remark this answers is not in the document]
Comment (martynov-alex) [reply pubspec.yaml:18]: fixed

File: pubspec.yaml (hunk 3 above)
Comment (martynov-alex) [reply pubspec.yaml:2]: fixed

File: pubspec.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (martynov-alex) [reply pubspec.yaml:18]: But there is a problem https://dart.dev/tools/pub/dependencies#dev-dependencies

If I move golden_toolkit to dev_dependencies I will need to add golden_toolkit to my app. 
So I guess we must leave it as is.

**Review by maxim-sysoev — APPROVED** [review]

**Pull request merged**